internal/splunkconfig/config/indexarchiverprovider_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 Splunk, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import "testing"
+
+func TestIndexArchiver_validate(t *testing.T) {
```

Comment:
Did you also mean to update this to `TestIndexArchiverProvider_validate` ?

Reply:
I can't tell if the intention is to use this for any other types that may be added in the future or if this is intended to be unique to `IndexArchiverProvider`